exif/exif.go

```diff
@@ -95,9 +96,10 @@ func loadSubDir(x *Exif, ptr FieldName, fieldMap map[uint16]FieldName) error {
 
 // Exif provides access to decoded EXIF metadata fields and values.
 type Exif struct {
-	Tiff *tiff.Tiff
-	main map[FieldName]*tiff.Tag
-	Raw  []byte
+	Tiff    *tiff.Tiff
+	main    map[FieldName]*tiff.Tag
+	Raw     []byte
+	Comment string
```

Comment:
How about a doc-string here describing what Comment is?  (I should also probably add doc comments for the rest of those fields too eh?